azure/src/test/java/sunstone/azure/armTemplates/di/AzPgSqlTests.java

```diff
@@ -10,45 +10,41 @@
 import sunstone.inject.Hostname;
 
 import static org.assertj.core.api.Assertions.assertThat;
-import static sunstone.azure.armTemplates.di.AzVmTests.group;
 
 
 @WithAzureArmTemplate(parameters = {
         @Parameter(k = "username", v = AzPgSqlTests.PGSQL_USER),
         @Parameter(k = "password", v = AzPgSqlTests.PGSQL_PASSWORD),
         @Parameter(k = "serverName", v = AzPgSqlTests.PGSQL_NAME),
 },
-        template = "sunstone/azure/armTemplates/posgresql.json", group = group, perSuite = true)
```

Comment:
Can we have unique test class specific group? Like in `MultipleAzArmTemplatesRepetableTest`.